src/EnergyPlus/AirflowNetworkBalanceManager.cc

```diff
@@ -2910,6 +2924,175 @@ namespace AirflowNetworkBalanceManager {
 			}
 		}
 
+		// Read Ourdoor Airflow object
+		CurrentModuleObject = "AirflowNetwork:Distribution:Component:OutdoorAirFlow";
+		NumOfOAFans = GetNumObjectsFound( CurrentModuleObject );
+		if ( NumOfOAFans > 1 ) {
+			ShowSevereError( RoutineName + "More " + CurrentModuleObject + " are found. Currently only one( \"1\") " + CurrentModuleObject + " object per simulation is allowed when using AirflowNetwork Distribution Systems." );
```

Comment:
The IDD memos for these objects should state that only one can be used.  And you could also tag these with \unique-object